cvat-ui/src/reducers/projects-reducer.ts

```diff
@@ -206,6 +209,50 @@ export default (state: ProjectsState = defaultState, action: AnyAction): Project
                 },
             };
         }
+        case ProjectsActionTypes.BACKUP_PROJECT: {
+            const { projectId } = action.payload;
+            const { backups } = state.activities;
+
+            return {
+                ...state,
+                activities: {
+                    ...state.activities,
+                    backups: {
+                        ...backups,
+                        ...Object.fromEntries([[projectId, true]]),
+                    },
+                },
+            };
+        }
+        case ProjectsActionTypes.BACKUP_PROJECT_FAILED:
+        case ProjectsActionTypes.BACKUP_PROJECT_SUCCESS: {
+            const { projectID } = action.payload;
+            const { backups } = state.activities;
+
+            delete backups[projectID];
```

Comment:
You omit this property in line 238. Do not need to change current object according to best practices.